### PR TITLE
Update base Biocontainers image

### PR DESCRIPTION
**Pull request by rspreafico**
This PR refers to [BioContainers issue 148](https://github.com/BioContainers/mulled/issues/148). Specifically, `procps` is added for Nextflow compatibility. 

Alpine Linux was used instead of BusyBox because of [BusyBox' deprecation warning](https://github.com/progrium/busybox). Bash is available through Alpine Linux' package manage `apk` and is set as the default shell.

The image name `bgruening/busybox-bash:0.2` was maintained for compatibility with the previous base image. However, it seems better to sensibly rename the base image, and make sure that derivatives refer to the new base image name.

Please see below size comparisons: Alpine Linux vs BusyBox vs Ubuntu:
```
$ docker images
REPOSITORY               TAG                 IMAGE ID            CREATED             SIZE
bgruening/busybox-bash          0.2              5dc8d3ca7b75        9 seconds ago       8.49MB
gliderlabs/alpine        3.6                 40b6e22d66ca        4 weeks ago         3.97MB
ubuntu                   16.04               747cb2d60bbe        2 months ago        122MB
bgruening/busybox-bash   0.1                 3d974f51245c        20 months ago       6.73MB
```

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,10 @@
-FROM progrium/busybox:latest
+FROM gliderlabs/alpine:3.6
 
-MAINTAINER Bjoern Gruening <bjoern.gruening@gmail.com>
+LABEL author="Roberto Spreafico <roberto.spreafico@gmail.com>"
```
Comment (bgruening) [inline Dockerfile:3]: Can we put this into the readme file?

File: Dockerfile [hunk 2]
```diff
@@ -1,6 +1,10 @@
...
+LABEL maintainer="Bjoern Gruening <bjoern.gruening@gmail.com>"
+
+RUN apk add --no-cache procps bash
+
+SHELL ["/bin/bash", "-c"]
```
Comment (bgruening) [inline Dockerfile:8]: I'm a little hesitant to change the default for so many images. Do you really need this?

**Review by bgruening — COMMENTED** [review]
This looks great, thanks. Two comments inline.

**Comment by bgruening** [comment]
As soon as this is merged, I will trigger a new build on the old repo, create a new repo and update the build-chain to use the new image `alpine-bash:0.2`. Is this ok for you?

Thanks again!

File: Dockerfile (hunk 1 above)
Comment (rspreafico) [reply Dockerfile:3]: Sorry for the delay, I missed the notification. Sure thing, and I do not have strong feelings about retaining this information anyway, it's a very simple image. Feel free to delete.

File: Dockerfile (hunk 2 above)
Comment (rspreafico) [reply Dockerfile:8]: We do need `procps` for Nextflow compatibility - this was the original reason triggering the overhaul of Biocontainers' base image. `bash` is added as it was included in the previous image too. As for the default shell, I don't have very strong opinions there. I cannot think at anything running on `sh` that wouldn't run on `bash` as well, but I understand the hesitation.

**Comment by rspreafico** [comment]
That would be great. As soon as you do that, I will notify the Nextflow crew about full compatibility between Nextflow and Biocontainers. That is a great achievement! Thanks for being open to this @bgruening !

File: Dockerfile (hunk 2 above)
Comment (bgruening) [reply Dockerfile:8]: I was only talking about the bin/bash here :)

**Pull request merged**